.github/workflows/rust.yml

```diff
@@ -23,14 +23,40 @@ jobs:
     - name: Install
       run: rustup update stable
     - name: Rust Cache
-      uses: Swatinem/rust-cache@v1.3.0
-    - name: Format
-      run: cargo fmt -- --check
-    - name: Run clippy
-      run: cargo clippy --all-targets --all-features --verbose -- -D warnings
+      uses: Swatinem/rust-cache@v1
     - name: Build
       run: cargo build --verbose
     - name: Run tests
       run: cargo test --verbose
     - name: Validate config
       run: cargo run config example.config.toml
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          profile: minimal
+          components: rustfmt, clippy
+          override: true
+
+      - uses: Swatinem/rust-cache@v1
+        with:
+          cache-on-failure: true
+
+      - name: cargo fmt
+        uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+          args: --all --check
+
+      - name: cargo clippy
+        uses: actions-rs/clippy-check@v1
+        with:
+          args: --all --all-features -- -D warnings
```

Comment:
I'm fine to drop `--verbose` although I see that `--all` is a deprecated option?

I think we can use `--all-targets` and get the same effect?

Reply:
right, also no needed since we don't have non default members